installing/installing-fips.adoc

```diff
@@ -14,19 +14,68 @@ For more information about the NIST validation program, see link:https://csrc.ni
 
-For more information about configuring FIPS mode on {op-system-base}, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening[Installing the system in FIPS mode].
+For more information about configuring FIPS mode on {op-system-base}, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode].
 ====
 
 For the {op-system-first} machines in your cluster, this change is applied when the machines are deployed based on the status of an option in the `install-config.yaml` file, which governs the cluster options that a user can change during cluster deployment. With {op-system-base-full} machines, you must enable FIPS mode when you install the operating system on the machines that you plan to use as worker machines.
 
 Because FIPS must be enabled before the operating system that your cluster uses boots for the first time, you cannot enable FIPS after you deploy a cluster.
 
+[id="installation-obtaining-fips-installer-oc_{context}"]
+== Obtaining a FIPS-capable installation program using `oc adm extract`
+
+{product-title} requires the use of a dynamically-linked installation binary to install a cluster in FIPS mode. You can obtain this binary by extracting it from the release image using `oc`. After you have obtained the dynamically-linked binary, you proceed with the cluster installation, replacing all instances of the `openshift-install` binary with `openshift-installer-fips`.
+
+.Prerequisites
+
+* You have installed the `oc` command line utility with version 4.16 or newer.
+* You have downloaded the installation program.
+
+.Procedure
+
+. Extract the dynamically-linked binary from the installation program by running the following command:
++
+[source,terminal]
+----
+$ oc adm release extract --command=openshift-installer-fips
+----
++
+. Proceed with cluster installation, replacing all instances of the `openshift-install` command with `openshift-installer-fips`.
+
+[id="installation-obtaining-fips-installer-mirror_{context}"]
+== Obtaining a FIPS-capable installation program using the public OpenShift mirror
+
+{product-title} requires the use of a dynamically-linked installation binary to install a cluster in FIPS mode. You can obtain this binary by downloading it from the public OpenShift mirror. After you have obtained the dynamically-linked binary, you proceed with the cluster installation, replacing all instances of the `openshift-install` binary with `openshift-baremetal-installer`.
+
+.Prerequisites
+
+* You have access to the internet.
+
+.Procedure
+
+. Download the `openshift-install-rhel9-amd64.tar.gz` file from https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.16/.
```

Comment:
```suggestion
. Download the `openshift-install-rhel9-amd64.tar.gz` file from https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.16/.
```

Maybe we should just link to the actual file rather than a directory?